### PR TITLE
support alibaba_tongyi stream output

### PR DESCRIPTION
**Pull request by sinajia**
Fixes # (issue)

### CODE REVIEW

**Review by bracesproul — COMMENTED** [review]
lgtm, could you add a test too? Thanks for doing this!

**Comment by sinajia** [comment]
fix type error, add a test

File: libs/langchain-community/src/chat_models/alibaba_tongyi.ts [hunk 1]
```diff
@@ -6,10 +7,12 @@ import {
   AIMessage,
   type BaseMessage,
   ChatMessage,
+  AIMessageChunk,
 } from "@langchain/core/messages";
 import { type ChatResult } from "@langchain/core/outputs";
-import { type CallbackManagerForLLMRun } from "@langchain/core/callbacks/manager";
```
Comment (jacoblee93) [inline libs/langchain-community/src/chat_models/alibaba_tongyi.ts:11]: I think you still need this type?

**Comment by jacoblee93** [comment]
Thank you! This will be released in about a week along with 0.2 - let me know if you need it cherry picked sooner.

**Pull request merged**